src/models/Article.php

```diff
@@ -174,8 +190,10 @@ public function ngRestAttributeGroups()
     public function ngRestScopes()
     {
         return [
-            [['list'], ['cat_id', 'title', 'timestamp_create', 'image_id']],
-            [['create', 'update'], ['cat_id', 'title', 'teaser_text', 'text', 'timestamp_create', 'timestamp_display_from', 'is_display_limit', 'timestamp_display_until', 'image_id', 'image_list', 'file_list']],
+            [['list'], ['id', 'cat_id', 'alias', 'title', 'timestamp_create', 'image_id']],
+            // `create` has additional permission to provide `id` for cases when administrator neews to move articles from other blog-modules
+            [['create'], ['id', 'alias', 'cat_id', 'title', 'teaser_text', 'text', 'timestamp_create', 'timestamp_display_from', 'is_display_limit', 'timestamp_display_until', 'image_id', 'image_list', 'file_list']],
```

Comment:
I think we should stick to all other modules here, entering the id of an PK AI field should be avoided.

Reply:
I've added `id` because I want to actually move one of my websites completely to LUYA, and for that I needed a way to provide article ID (for `insert` scenario only). We could have an option on Module level to allow/disable providing of ID when creating an article.

Reply:
We can not do this here i think. If you want to import news, why not just create a command very quickly? you could even connect directly to the database. So i don't think its a good approach to just add this element, which makes not that much sense of 99.5% of uses cases.

Reply:
@nadar good point!